### DISCUSSION TITLE
Skipping through empty directories

### DISCUSSION THREAD

**Discussion by RoinujNosde** [community/community#18379] [Mobile]
Hi,
Please make it possible to skip through empty directories like on the web version (see image).

![image](https://user-images.githubusercontent.com/6441328/173205896-c4017278-bee3-426c-9653-a937e44fe3cc.png)

**Comment by eliperkins** [comment]
Hey @RoinujNosde, thanks for this feedback!

Skipping empty directories sounds like a great idea that would make browsing codebases on mobile much more efficient.

I've shared your notes about this with our team on an internal tracking issue. I'll keep you posted on this feature when it comes!

**Reply by ParkerM** [reply]
Any updates on this? The mobile app has become impressively valid and usable as an alternative to the desktop web client, and I find myself switching between both interfaces with little friction. The repo file browser seems to have really fallen behind the web client, however. There have been some great QoL improvements for browsing in the web client, with features like the tree view that make it rival a desktop/IDE file browser. Visiting this view in the mobile app, however, takes me to a jarring interface  that is clumsy to navigate and is in dire need of some TLC. It feels out-of-place and poorly represents the otherwise excellent usability of the mobile app.
